test/test_fake_tensor.py

```diff
@@ -631,6 +634,22 @@ def contains_type(type: torch._C.Type, maybe_contained_type: torch._C.Type):
     )
 
 
+class FakeTensorOpInfoTest(TestCase):
+    @ops(custom_op_db, dtypes=OpDTypes.any_one)
+    def test_fake(self, device, dtype, op):
+        sample_inputs_itr = op.sample_inputs(device, dtype, requires_grad=False)
+        for sample_input in sample_inputs_itr:
+            args = (sample_input.input,) + sample_input.args
+            kwargs = sample_input.kwargs
+        with torch._subclasses.CrossRefFakeMode():
+            try:
+                op(*args, **kwargs)
+            # This is allowed from a FakeTensor implementation. Dynamo knows
+            # how to graph break on them.
+            except DynamicOutputShapeException:
```

Comment:
Is the point of these tests just to confirm that every custom op in the db either doesn't raise an exception, or raises a DynamicOutputShapeException? Is it worth tagging the customOpInfo's that will, and asserting that only the tagged ones will? (Not necessary if you don't think it's worth implementing).

Reply:
> Is the point of these tests just to confirm that every custom op in the db either doesn't raise an exception, or raises a DynamicOutputShapeException?

Yes. In addition to those, the CrossRef also checks that sizes/strides/devices are correct between the FakeTensor rule and actually executing the operators (for the ones that do not raise DynamicOutputShapeException)

> Is it worth tagging the customOpInfo's that will, and asserting that only the tagged ones will?

This is a good point. I'll add a simple tagging mechanism (i.e. a list somewhere) to do this.